web/viewer.js

```diff
@@ -1686,7 +1686,7 @@ function webViewerInitialized() {
   }
 //#endif
 
-//#if !B2G && !CHROME
+//#if !(FIREFOX || MOZCENTRAL) && !B2G && !CHROME
```

Comment:
Since this preprocessor line is identical to `GENERIC`, why not just move this code:
```javascript
if (file) {
  PDFViewerApplication.open(file, 0);
}
```
to after [line 1686](https://github.com/mozilla/pdf.js/pull/5951/files#diff-dd42e9b2d2a652b8e312efa567698aa6R1686) instead?